### PR TITLE
ports nanoweave carpets from NSV

### PR DESCRIPTION
**Pull request by triplezeta**
## About The Pull Request

yo i probably should have asked for permission to do this beforehand but oh well

craft with plasteel

## Why It's Good For The Game

a less regal carpet to use for carpetty things

i should probably resprite these because they dont look that great up close now that i think of it

## Changelog
:cl: triplezeta & untrius
add: nanoweave carpets
/:cl:

### CODE REVIEW

File: code/__DEFINES/icon_smoothing.dm [hunk 1]
```diff
@@ -126,3 +126,12 @@ DEFINE_BITFIELD(smoothing_flags, list(
...
 #define MAX_S_OBJ SMOOTH_GROUP_CLEANABLE_DIRT //Always match this value with the one above it.
+
+//WS edit: ports Nanoweave carpet from NSV
+
+#define SMOOTH_GROUP_CARPET_NWDARK(70)
+#define SMOOTH_GROUP_CARPET_NWRED(71)
+#define SMOOTH_GROUP_CARPET_NWBLUE(72)
+#define SMOOTH_GROUP_CARPET_NWPURPLE(73)
+#define SMOOTH_GROUP_CARPET_NWORANGE(74)
+#define SMOOTH_GROUP_CARPET_NWBEIGE(75) //WS end
```
Comment (MarkSuckerberg) [inline code/__DEFINES/icon_smoothing.dm:137]: Move to modularized file

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]

**Comment by untrius** [comment]
Here's what they look like:

image

**Comment by untrius** [comment]
I didn't modularize the new smoothing group defines, because any change to them requires a change to the existing smoothing group file.

**Comment by untrius** [comment]
I'm closing this for now.  But I want it on the record that it works, in case anyone ever decides to resprite these.

**Pull request closed without merging**